### PR TITLE
Q025, Q038

### LINKED ISSUE #192: Validation (2017): Q038 (Loan Amount)
**Edit #:** Q038
**Type:** Quality
**Category:** Loan Application Register (only)
**Column:** Loan Amount

If lien status = 3, then loan amount should be ≤ $100 ($100 thousand).

Additional resources:
* [FFIEC's 2016 HMDA File Spec](https://www.ffiec.gov/hmda/pdf/spec2016.pdf)
* [FFIEC's 2016 HMDA Edits](https://www.ffiec.gov/hmda/pdf/edit2016.pdf)
* [CFPB's 2017 HMDA File Spec](http://www.consumerfinance.gov/hmda/static/for-filers/2017/2017-HMDA-File-Specifications.pdf)

### LINKED ISSUE #193: Validation (2017): Q025 (Loan Amount)
**Edit #:** Q025
**Type:** Quality
**Category:** Loan Application Register (only)
**Column:** Loan Amount

If loan purpose = 1 and property type = 1, then loan amount should be > $10 thousand.

Additional resources:
* [FFIEC's 2016 HMDA File Spec](https://www.ffiec.gov/hmda/pdf/spec2016.pdf)
* [FFIEC's 2016 HMDA Edits](https://www.ffiec.gov/hmda/pdf/edit2016.pdf)
* [CFPB's 2017 HMDA File Spec](http://www.consumerfinance.gov/hmda/static/for-filers/2017/2017-HMDA-File-Specifications.pdf)

### PR DESCRIPTION
**Pull request by nickgrippin**
Closes #192 
Closes #193

### CODE REVIEW

File: validation/src/main/resources/application.conf [hunk 1]
```diff
@@ -17,12 +17,20 @@ hmda.validation.quality {
...
   }
   Q037 {
-    loan.amount=250
-    loan.amount=${?Q037_LOAN_AMOUNT}
+    loan.amount = 250
+    loan.amount = ${?Q037_LOAN_AMOUNT}
+  }
+  Q038 {
+    loan.amount = 100
+    loan.amount = ${?Q035_LOAN_AMOUNT}
```
Comment (lgdean) [inline validation/src/main/resources/application.conf]: I notice some back-and-forth about how to format this file. We might want to pick a standard way, and stick with it. (I don't have an opinion of which way.)

Comment (nickgrippin) [inline validation/src/main/resources/application.conf]: Sorry, that was just the autoformatting in IntelliJ

**Comment by schbetsy** [comment]
LGTM

File: validation/src/main/resources/application.conf (hunk 1 above)
Comment (lgdean) [inline validation/src/main/resources/application.conf]: Is that something we can configure? (Or, do we wanna standardize on doing whatever it does, as long as it's not too icky?) Not a blocker for this PR, but might help avoid this kind of noise.

File: validation/src/main/resources/application.conf [hunk 2]
```diff
@@ -25,4 +29,8 @@ hmda.validation.quality {
     loan.amount=250
     loan.amount=${?Q037_LOAN_AMOUNT}
   }
+  Q038 {
+    loan.amount=100
+    loan.amount=${?Q035_LOAN_AMOUNT}
```
Comment (kgudel) [inline validation/src/main/resources/application.conf]: Typo in name here `Q035` instead of `Q038`.

**Pull request merged**